src/constants/settings.ts

```diff
@@ -23,3 +23,5 @@ export const networkConfig: NetworkConfig = {
     stellarExpertTxUrl: "https://stellar.expert/explorer/public/tx/",
   },
 };
+
+export const qrCodeUrl = "https://stellar-wallet-apis.fnando.now.sh/api";
```

Comment:
@quietbits I took this from Mobile-Wallet, but not sure if we should be using here as well or a different one?